### ISSUE TITLE
Using font families with space and numbers at the end

### ISSUE THREAD

**Issue by asturur** [bramstein/fontfaceobserver#70]
I stepped in a bug in wich loading:

```
const observer = new fontfaceobserver(fontName, { weight, style });
observer.check();
```

makes the promise fails immediately

The font name are "Exo 2" and "Slabo 27px"  i guessed that those numbers could confuse the font parser and i tried

```
const observer = new fontfaceobserver(`'${fontName}'`, { weight, style });
observer.check();
```

And everything went ok.

Other font families with space in the middle does not create any problem.

I was asking myself if this is considered a bug or if the developer care to properly quote its font families.

If this is a bug i can send a PR solving it?

**Comment by asturur** [comment]
Adding a note:  using single quote as i did breaks ie, edge and safari. Using double quote looks like  ok for all of them.

**Comment by cadonau** [comment]
I think I came across the same issue. Using “Neue Frutiger 1450 W04” as font-family name results in immediate failure: `Uncaught (in promise) g {family: "Neue Frutiger 1450 W04", style: "normal", weight: "normal", stretch: "normal"}`. Prefixing “1450” (everywhere) resolves the issue.

**Comment by bramstein** [comment]
This should be fixed in v2.0.3. Thanks for reporting the issue!
